test/test-timer.c

```diff
@@ -161,6 +161,7 @@ TEST_IMPL(timer_init) {
 
   ASSERT(0 == uv_timer_init(uv_default_loop(), &handle));
   ASSERT(0 == uv_timer_get_repeat(&handle));
+  ASSERT_LE(0, uv_timer_get_due_in(&handle));
```

Comment:
Should the new assertions in this file use the `ASSERT_UINT64_*` family of assertions since we're dealing with `uint64_t`s? The current assertions use `int64_t`s.

I think you might be able to drop the casts from the assertions, as there are internal casts in the assertion macros.